docs/website/generator-errors.md

```diff
@@ -198,6 +198,10 @@ This usually indicates a bug in Xamarin.iOS/Xamarin.Mac; please [file a bug repo
 
 ### <a name='BI1066'/>BI1066: Unsupported return type '{type}' in {method}.
 
+### <a name='BI1067'/>BI1067: BindAs attribute of the selector \"{selector}\" is missing '[]' in the definition.
```

Comment:
I'd say something like: `The BindAs type for the member "{type}.{member}" must be an array when the member's type is an array.`

This could also benefit from some text that explains the problem (it doesn't have to be more than a bad example and then a good/corrected example).